test/output.js

```diff
@@ -0,0 +1,129 @@
+import {serial as test} from 'ava';
+import hookStd from 'hook-std';
+import {alfy} from './_utils';
+
+const itemWithMod = {
+	title: 'unicorn',
+	arg: '🦄',
+	env: {fabulous: true},
+	mods: {
+		alt: {
+			title: 'Rainbow',
+			arg: '🌈',
+			env: {
+				color: 'myriad'
+			}
+		}
+	}
+};
+
+const m = alfy();
+const hook = cb => {
```

Comment:
Return a promise instead of a callback, this way you can just use `async/await` in your tests.